### ISSUE TITLE
Support for Prettier 3.x

### ISSUE THREAD

**Issue by coycoylaniba** [IanVS/prettier-plugin-sort-imports#112]
**Is your feature request related to a problem?**
Not that big

**Describe the solution you'd like**
@ianvs/prettier-plugin-sort-imports 4.0.2
  └── ✕ unmet peer prettier@2.x: found 3.0.0

**Describe alternatives you've considered**
I have nothing.

**Additional context**
![image](https://github.com/IanVS/prettier-plugin-sort-imports/assets/47078705/f7b981dc-8697-436a-841a-d338e9bfe607)

**Comment by darenmalfait** [comment]
When using prettier 3 I'm actually not able to run it. Getting a lint error when using this as a plugin.

```
SLint: 8.44.0

TypeError: Cannot set property message of  which has only a getter
    at Linter._verifyWithoutProcessors
```

**Comment by krisgrm** [comment]
> 

could be related? https://stackoverflow.com/questions/76624993/prettier-3-0-0-typeerror-prettier-resolveconfig-sync-is-not-a-function

**Comment by IanVS** [comment]
@darenmalfait that looks like an ESLint error.  Do you have a reproduction you can share?

**Comment by IanVS** [comment]
We've released [4.1.0](https://github.com/IanVS/prettier-plugin-sort-imports/releases/tag/v4.1.0) with full support for prettier 3.  If anyone has a problem, please open a new issue with as much detail as you can, and ideally a way to reproduce it.

**Comment by darenmalfait** [comment]
My issue is fixed with 4.1.0. 👍
